gender_novels/deployment/static/markdowns/gender_novels_overview.md

```diff
@@ -3,26 +3,29 @@
 ## Our Goals
 
 As the first project of the lab, we are very excited to begin our research in the 
-field of gender studies. The use of gender has changed over the centuries in the use of our 
-literature.  We are directing our research at novels written by authors as famous as Jane Austen
-and others that are much more obscure. We have been 
+field of gender studies. The use of gender has changed over the centuries, not only in our 
+culture, but also in the use of our literature.  We are directing our research at novels written 
+by authors as famous as Jane Austen and others that are much more obscure. By looking at the history
+of our books, we hope to be able to study the ever-adapting and changing view on gender by writers
+all around the globe in the nineteenth and twentieth centuries.
 
+![Books](http://ichef.bbci.co.uk/wwfeatures/wm/live/1280_640/images/live/p0/2v/dp/p02vdpfn.jpg 
+"Books")
 
 
 ## The Lab
```

Comment:
As this is a website only for this project, maybe save this for insertion into the lab homepage?